### PR TITLE
Fix protocols faq

### PR DESCRIPTION
**Pull request by ranand**
### Proposed changes
- We shipped Duplicate Tracking Plans and Libraries in Q3 (Public Beta) and moving to GA in Q4. 
- PRD: https://docs.google.com/document/d/1b6jr7CsBk_YNJAvcATNOVi8bgCQi6P0ZtlVWkL5GxAk/edit#heading=h.vv9txf1zeg8
- In this PR, I propose that we fix the FAQ for Protocols to ensure we call out the ability to copy/duplicate tracking plans.

### Merge timing
- Merge on 10/24/2022.

### CODE REVIEW

File: src/protocols/faq.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The Tracking Plan to Source relationship is a one-to-many relationship. This mea
 
 ### Can I duplicate a Tracking Plan in the Segment UI?
 
-You cannot currently duplicate Tracking Plans in the Segment web app. Instead, Segment recommends using the [Tracking Plan API](/docs/protocols/apis-and-extensions/) to copy the underlying JSON schema from one Tracking Plan to another.
+You can duplicate Tracking Plans in the Segment web app. Here are the instructions to [copy a tracking plan](/docs/protocols/tracking-plan/create/#copy-a-tracking-plan). You can also use the [Tracking Plan API](/docs/protocols/apis-and-extensions/) to copy the underlying JSON schema from one Tracking Plan to another.
```
Comment (stayseesong) [inline src/protocols/faq.md:41]:
```suggestion
Yes, you can duplicate Tracking Plans in the Segment web app by following the [instructions to copy a tracking plan](/docs/protocols/tracking-plan/create/#copy-a-tracking-plan). You can also use the [Tracking Plan API](/docs/protocols/apis-and-extensions/) to copy the underlying JSON schema from one Tracking Plan to another.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/protocols/faq.md (hunk 1 above)
Comment (stayseesong) [reply src/protocols/faq.md:41]: @ranand is the feature to duplicate tracking plans currently in beta or did it already go to GA. If it's in public beta, we'll need to call that out here. When's it going to GA?

**Comment by stayseesong** [comment]
@ranand, just to clarify, the feature is going to GA on 10/31?

**Review by stayseesong — APPROVED** [review]

**Comment by ranand** [comment]
> @ranand, just to clarify, the feature is going to GA on 10/31?

It is going GA on 10/31, yes.

**Comment by ranand** [comment]
@stayseesong - fixed the date to reflect the correct date. It is going GA on 10/25.

**Comment by stayseesong** [comment]
@ranand ok, will get that merged in for you by eod today

**Pull request merged**